object/search.go

```diff
@@ -160,6 +161,12 @@ func (f *SearchFilter) Operation() SearchMatchType {
 	return f.op
 }
 
+// IsNonAttribute checks if SearchFilter is non-attribute: such filter is
+// related to the particular property of the object instead of its attribute.
+func (f SearchFilter) IsNonAttribute() bool {
```

Comment:
what gonna happen with that method if some property is moved to the header fields?

Reply:
if completely moved (next protocol major), then return `true` as expected. If with backward compatibility, then still `false` cuz attribute will be kept (that's why i mention `particular property` instead of `header field`)